shell/install-example.js

```diff
@@ -12,8 +12,9 @@ cd(example);
 // Install node modules
 exec(`yarn install`);
 
-// Install jspm packages
-exec(`jspm install --log warn -y`);
+// Fix a bug where Webpack does not dedupe node_modules in linked packages
+exec(`rm -fr ./node_modules/react-helmet`);
+exec(`ln -sfv ./react-wildcat-handoff/node_modules/react-helmet ./node_modules/react-helmet`);
```

Comment:
Ah I'm guessing why this is why we needed to add the eslint ignore unresolved line in the above file